hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassCluster.java

```diff
@@ -144,6 +149,11 @@ public static synchronized Session getSession(boolean overwrite) throws Exceptio
             String nodes = AlertProperties.getProperty(ALERTS_CASSANDRA_NODES, ALERTS_CASSANDRA_NODES_ENV, "127.0.0.1");
             int attempts = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_RETRY_ATTEMPTS, "5"));
             int timeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_RETRY_TIMEOUT, "2000"));
+            int connTimeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_CONNECT_TIMEOUT,
+                    ALERTS_CASSANDRA_CONNECT_TIMEOUT_ENV, "5000"));
+            int readTimeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_READ_TIMEOUT,
+                    ALERTS_CASSANDRA_READ_TIMEOUT_ENV, "12000"));
```

Comment:
Are these the datastax defaults as well, or are you setting them differently from out-of-box settings?  Perhaps by default we should not override the driver?